project/Release.scala

```diff
@@ -26,12 +26,12 @@ object Release {
       runTest,
       commitReleaseVersion,
       tagRelease,
-      ReleaseStep(releaseStepTask(publish in docker)),
+      ReleaseStep(releaseStepTask(docker / publish)),
```

Comment:
Have you tried building the image locally like this? I'd be a bit surprised if this works using distroless and sbt-native-packager together. sbt-native-packager tends to create quite complicated shell scripts as entrypoints but distroless images have no shell.

Reply:
You're right. For this PR I've opted to go for https://hub.docker.com/_/eclipse-temurin 's alpine image until we can explore using distroless for this project.

Reply:
you've tested it works right? just wanna make sure

Reply:
Yeah works fine, we do need to add bash but it starts up as expected. The Kafka not subscribed we see in the current release if there's no topics available at startup as we don't have any retry policies

![image](https://user-images.githubusercontent.com/28694181/169834645-462164de-415c-469e-bab3-9ac1a90dfcd9.png)